internal/slice/slice.go

```diff
@@ -0,0 +1,16 @@
+package slice
+
+const (
+	Wildcard    = "*"
+	X509Subject = "x509.subject"
```

Comment:
If these are used only  in `trustpolicy.go` should we move them to `trustpolicy.go` file?

Reply:
They will also be used in the verifier package (this part is will be in PR #186 where verification package is refactored). If we put them into trustpolicy package, then they will become exported.